This file is part of MXE.
See index.html for further information.

This patch was taken from Gentoo:
https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/coin/files/coin-3.1.3-gcc-4.7.patch?id=17d7c853393ff83e3422e48e9ad2810f23889bbf

--- coin3-3.1.3.orig/include/Inventor/SbBasic.h
+++ coin3-3.1.3/include/Inventor/SbBasic.h
@@ -24,6 +24,7 @@
  *
 \**************************************************************************/
 
+#include <Inventor/C/errors/debugerror.h>
 #include <Inventor/C/basic.h>
 
 /* ********************************************************************** */
